src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
```

Comment:
Using wildcard imports like '*' is discouraged as it can lead to naming conflicts and make the code less readable. It's better to import specific classes.